packages/server-core/src/media/file-browser/file-browser.class.ts

```diff
@@ -264,7 +264,7 @@ export class FileBrowserService
       }
     }
 
-    validateSceneName(data.path)
+    if (data.type === 'scene') validateSceneName(data.path)
```

Comment:
we need to validate all files. which files are being found as illegal?

Reply:
Thumbnails being uploaded. Underscores are allowed in files but not in our scenes names.